### DISCUSSION TITLE
Option to prevent the UI from rounding

### DISCUSSION THREAD

**Discussion by gforcedev** [ananthakumaran/paisa#66] [Ideas]
I've been enjoying using paisa, but it's important to me to be able to see exactly (instead of being rounded to the nearest <currency>) what's been spent on different things. It would be great to have the option to stop the UI rounding everything

**Comment by ananthakumaran** [comment]
Got it, my country's currency is way more inflated, so I normally don't see the need for decimals, but it makes sense for other currencies. Let me see if I can make the `precision` configurable. Btw, this will only cover places where I show the full number now, there are cases where I show shorthand form (like 100k) due to space constraints, which will continue to be the same.

I will get it done, might take some time though. I have a few core functions that handle the formatting, updating those should solve this.

**Reply by ananthakumaran** [reply]
new config named `display_precision` has been added to master https://github.com/ananthakumaran/paisa/commit/e575fe61f54dff2097992e72df7fb204b05e5fe4
